apps/devhub/views.py

```diff
@@ -1777,8 +1777,23 @@ def api_key(request):
         new_credentials = APIKey.new_jwt_credentials(request.user)
         log.info('new JWT key created: {}'.format(new_credentials))
 
+        send_key_change_email(request.user.email, new_credentials.key)
```

Comment:
ah nice! Didn't know about that, and it seems not many other people did ;) (maybe that's why most emails are sent using tasks?)